src/v2/ion/i18nTransformer.js

```diff
@@ -151,6 +151,7 @@ const I18N_OVERRIDE_MAPPINGS = {
   'oie.authenticator.duo.method.duo.verification_failed': 'oie.authenticator.duo.error',
 
   'idx.email.verification.required': 'registration.complete.confirm.text',
+  'tooManyRequests': 'oie.tooManyRequests',
```

Comment:
Is `yarn lint` passing on this PR? You may need to add `tooManyRequests` to `packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js` if it doesn't.

Reply:
it is passing. 🟢